### PR TITLE
7.x Documentation update preparing for 1.12 release

### PR DESCRIPTION
**Pull request by DiegoPino**
**JIRA Ticket**: None

# What does this Pull Request do?

Updates this module's documentation and moves attached media to its own relative folder.
Note: will create release version once this one is approved/corrected. (after lunch!)

# What's new?

- Updated the general setting screenshot 
- Mention some new options
- Mention that Solr is a soft dependency and explain why?
- Removed the red box around the JAIL option in the general settings screenshot. Never understood why we should mark that one only since we have at least 8 more to explain. Better to general in that case
- Changed some wording

# How should this be tested?

Proofread, check Screenshot, see if my words make sense, correct me if i'm wrong

# Interested parties
@manez @bryjbrown @bondjimbond

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,27 +10,38 @@ This module requires the following modules/libraries:
...
-ability as outlined below.
+ability using Javascript by checking 'Use alternative, alternative dsiplay for compounds' as shown below.
+
+
+![Configuration](docs/compound-basic-settings.png)
+
+### Solr Integration
 
-![Configuration](https://user-images.githubusercontent.com/25011926/39889778-d1a91aca-5466-11e8-8eb1-1978cac81104.png)
+If the Solr Search Module is enabled, and additional Compound Member Query Option and a new 'Solr Backend' Menu tab will be activated at Administration » Islandora » Solution Pack Configuration » Compound Object Solution Pack (admin/islandora/solution_pack_config/compound_object/solr)
```
Comment (rosiel) [inline README.md]: Thanks Diego! Did you mean "an additional" instead of "and additional"?

Comment (DiegoPino) [reply README.md]: Are you mentalist 😄 ? Yes!! Fixing after lunch, thanks so much for proofreading!!

**Review by rosiel — APPROVED** [review]

**Comment by rosiel** [comment]
Super, Diego (both with comma and without)! Since it's Doc'n, I'm not waiting 24h... but I will wait for the release pull so i can merge simultaneously.

**Comment by DiegoPino** [comment]
Amazing, Rosie (tokenized and as a single phrase too!). Here is the release equivalent
https://github.com/Islandora/islandora_solution_pack_compound/pull/151

**Pull request merged**